graphql/me/mutation.ts

```diff
@@ -225,6 +225,8 @@ class BecomeStatePupilInput implements BecomeStatePupilData {
     gradeAsInt?: number;
 }
 
+const logger = getLogger('Me');
```

Comment:
To keep it consistent.
```suggestion
const logger = getLogger('Me Mutations');
```